src/BundleManager.js

```diff
@@ -175,19 +175,26 @@ class BundleManager {
     Logger.verbose(`\tLOAD: Attributes...`);
 
     const attributes = require(attributesFile);
-    let error = `\tAttributes file [${attributesFile}] from bundle [${bundle}]`;
     if (!Array.isArray(attributes)) {
+      let error = `\tAttributes file [${attributesFile}] from bundle [${bundle}]`;
       Logger.error(`${error} does not define an array of attributes`);
       return;
     }
 
+    this.addAttributesList(attributes);
+
+    Logger.verbose(`\tENDLOAD: Attributes...`);
+  }
+
+  addAttributesList(attributes) {
     for (const attribute of attributes) {
       if (typeof attribute !== 'object') {
         Logger.error(`${error} not an object`);
         continue;
       }
 
       if (!('name' in attribute) || !('base' in attribute)) {
+        let error = `\tAttributes file [${attributesFile}] from bundle [${bundle}]`;
```

Comment:
Since you moved error variable inside of if(..), use const
```suggestion
        const error = `\tAttributes file [${attributesFile}] from bundle [${bundle}]`;
```